bom/pom.xml

```diff
@@ -253,6 +253,31 @@
         <version>4.1.68.Final-SNAPSHOT</version>
         <classifier>osx-x86_64</classifier>
       </dependency>
+
+      <!-- Add netty-tcnative* as well as users need to ensure they use the correct version -->
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-tcnative</artifactId>
+        <version>${tcnative.version}</version>
```

Comment:
Where is `${tcnative.version}` being set? The pom says "keep in sync" but I don't see a property section here.

Reply:
parent pom.xml